src/Domain.jl

```diff
@@ -41,7 +41,7 @@ function ConstantTPDomain(;phase::E2,initialconds::Dict{X,X2},constantspecies::A
     #set conditions and initialconditions
     T = 0.0
     P = 0.0
-    y0 = zeros(length(phase.species)+1)
```

Comment:
I think this commit reverting integrateV got caught unintentionally, it should be deleted.

Reply:
Thanks for the comment. I dropped the reverting commit!